src/elements.cpp

```diff
@@ -1452,7 +1459,6 @@ namespace OpenBabel
           break;
         }
         break;
-      }
```

Comment:
This deletion is probably responsible for the failed tests.

Reply:
@fredrikw I have made the changes as per your suggestion. Please review.